router/core/websocket.go

```diff
@@ -463,13 +470,42 @@ func (h *WebSocketConnectionHandler) executeSubscription(ctx context.Context, ms
 	// This gets removed by WebSocketConnectionHandler.Complete()
 	h.subscriptions.Insert(msg.ID, cancel)
 
-	ctxWithOperation := withOperationContext(cancellableCtx, operation, h.clientInfo)
+	variablesCopy := make([]byte, len(operation.Variables))
```

Comment:
This is duplicated in order to freely modify the request context in both places.